lib/ansible/module_utils/remote_management/dellemc/dellemc_idrac.py

```diff
@@ -0,0 +1,97 @@
+# _*_ coding: utf-8 _*_
+
+#
+# Dell EMC OpenManage Ansible Modules
+# Version 1.0
+# Copyright (C) 2018 Dell Inc.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# All rights reserved. Dell, EMC, and other trademarks are trademarks of Dell Inc. or its subsidiaries.
+# Other trademarks may be trademarks of their respective owners.
+#
+
+from __future__ import (absolute_import, division,
+                        print_function)
+import tempfile
+import os
+__metaclass__ = type
+
+try:
+    from omsdk.sdkinfra import sdkinfra
+    from omsdk.sdkcreds import UserCredentials
+    from omsdk.sdkfile import FileOnShare, file_share_manager
+    from omsdk.sdkprotopref import ProtoPreference, ProtocolEnum
+    from omsdk.http.sdkwsmanbase import WsManOptions
+    HAS_OMSDK = True
+
+except ImportError:
+
+    HAS_OMSDK = False
+
+
+class iDRACConnection():
+    def __init__(self, module):
+        if HAS_OMSDK is False:
+            results = {}
+            results['msg'] = "Dell EMC OMSDK library is required for this module"
```

Comment:
This thing conflicts with msg return value being declared @ https://github.com/ansible/ansible/pull/46675/files#diff-62a00abb6ff3cd2c35f54b8b82436130R104

Reply:
updated the code block